fastn-core/src/http.rs

```diff
@@ -1016,3 +1016,14 @@ mod test {
         Ok(())
     }
 }
+
+static HEADER_KEY_PATTERN: once_cell::sync::Lazy<regex::Regex> =
+    once_cell::sync::Lazy::new(|| regex::Regex::new(r"\$header-(\w+)\$").unwrap());
```

Comment:
Don't use regex for this, write a split_once based parser.

Reply:
Okay